cmd/kubeadm/app/apis/kubeadm/v1beta3/defaults.go

```diff
@@ -181,23 +181,7 @@ func SetDefaults_BootstrapTokens(obj *InitConfiguration) {
 	}
 
 	for i := range obj.BootstrapTokens {
-		SetDefaults_BootstrapToken(&obj.BootstrapTokens[i])
-	}
-}
-
-// SetDefaults_BootstrapToken sets the defaults for an individual Bootstrap Token
-func SetDefaults_BootstrapToken(bt *bootstraptokenv1.BootstrapToken) {
```

Comment:
Although we removed `SetDefaults_BootstrapToken` without a deprecate phase, given that it's just a package replace, I think this change is OK.

/lgtm

Reply:
indeed, there is no other way without blocking the work on the api